spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -119,25 +120,13 @@
 			<dependency>
 				<groupId>com.google.cloud.sql</groupId>
 				<artifactId>mysql-socket-factory</artifactId>
-				<version>1.0.8</version>
-				<exclusions>
```

Comment:
Is this library no longer pulled in by them?

Reply:
They exclude it: https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/blob/28b266b13ec4a9b6204226a864453d3903095f7e/core/pom.xml#L29